agent/bin/agent.js

```diff
@@ -82,6 +83,10 @@ function start(env) {
                         exitHandler();
                     });
                 });
+
+                setInterval(() => {
+                    log.info("Heap statistics : %j", v8.getHeapStatistics());
+                }, 60000);
```

Comment:
Is printing the heap every minute, temporary?  And we can remove it once we know why it is getting so large?

Reply:
I'm inclined to keep it, though possible at a reduced frequency.  I'm not a node expert, but I couldn't find any external method to get the virtual machine's state at run time so was rather blind.  Having some heap statistics over time helps.